### PR TITLE
fix: fix BaseEntity equality test

### LINKED ISSUE #25: All the transient BaseEntities are equal
When two base entities are _transient_ (`id==null`), the equality test boils down to `this==obj || Objects.equals(null,null);` 
https://github.com/AppJars/commons-backend/blob/master/commons-model/src/main/java/com/appjars/saturn/model/BaseEntity.java#L51-L60

### PR DESCRIPTION
**Pull request by javier-godoy**
Close #25

### CODE REVIEW

File: commons-model/src/main/java/com/appjars/saturn/model/Identifiables.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Utility methods for {@link Identifiable} objects. 
+ * 
+ * @author Javier Godoy / Flowing Code
+ *
+ */
+public class Identifiables {
```
Comment (mlopezFC) [inline commons-model/src/main/java/com/appjars/saturn/model/Identifiables.java:31]: If this is a utility class, maybe it's better to call it IdentifiableUtils or IdentifiableUtilities

**Review by mlopezFC — COMMENTED** [review]
Only one comment, the rest LGTM

File: commons-model/src/main/java/com/appjars/saturn/model/Identifiables.java (hunk 1 above)
Comment (javier-godoy) [reply commons-model/src/main/java/com/appjars/saturn/model/Identifiables.java:33]: In this case, `Identifiables` does with `Identifiables` the same that `java.util.Objects` does with `java.lang.Object`:
https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#toString-java.lang.Object-
https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object-
https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#hashCode-java.lang.Object-

It was non-final and had a public constructor thought (fixed in the last commit)

Comment (mlopezFC) [reply commons-model/src/main/java/com/appjars/saturn/model/Identifiables.java:33]: Oh, I didn't know the source of the naming, it's ok, let's use that scheme.

**Review by mlopezFC — APPROVED** [review]
LGTM

**Pull request merged**